bin/scpipe/lib/sc/pipe.rb

```diff
@@ -69,37 +71,32 @@ def prepare_pipe
       end
 
       def run_pipe
-        @@pipeproc = Proc.new {
-          trap("INT") do
-            Process.exit
-          end
-          rundir = Dir.pwd
+        rundir = Dir.pwd
+        while @done==false do
           IO.popen("cd #{rundir} && #{SC.sclang_path.chomp} -d #{rundir.chomp} -i scvim", "w") do |sclang|
```

Comment:
Ruby does not seem to terminate the additional `cat` reliably, which can lead to `cat` process buildup in addition to the `ruby` and `sclang` process buildup. Stray cats!!! :)

Using the ruby internal File.open instead solves the problem, as ruby can certainly clean up reliably after _itself_.